### DISCUSSION TITLE
Faker bug - support for `enumObject` for Mikro-ORM Seeder. Needs upgrade Faker to v.8

### DISCUSSION THREAD

**Discussion by adamhemzal** [mikro-orm/mikro-orm#4981] [Q&A]
### Describe the bug

**EDIT: `faker` is removed in upcoming version `v6`  : https://mikro-orm.io/docs/next/upgrading-v5-to-v6#removed-dependency-on-faker-in-seeder-package**

I want to seed data into my `product` table. I have a `type` attribute which is `enum`. In [Faker](https://fakerjs.dev/api/helpers.html#enumvalue) documentation they recommend using `enumValue` to load data from enum. However, the current version of mikro-orm `5.9.4` probably does not have updated faker package, because I can't see `enumValue`. I might be mistaken, but I think the faker package is outdated and needs to be updated.

```text
Property 'enumValue' does not exist on type 'HelpersModule'.ts(2339)
```

Thank you for your help!

**Update: I found that the current version of faker in mikro-orm is `7.6.0`. So updating to v `8.0.0` should solve this bug**

I'm using:
- `@mikro-orm/core": "^5.9.4"` + `"@mikro-orm/nestjs": "^5.2.2"` + `"@nestjs/core": "^10.0.0",` + `"@mikro-orm/postgresql": "^5.9.4",`
- Windows10
- Node v18.17.1
- VSCode

### Reproduction

```typescript
// EProduct.ts

export enum EProduct {
 ITEM_A = 'item-a',
 ITEM_B = 'item-b',
}
```

```typescript
// ProductFactory.ts

import { Factory, Faker } from '@mikro-orm/seeder';
import { EProduct } from './enums';
import { Product } from './entities/Product';

export class ProductFactory extends Factory<Product> {
  model = Product;

  definition(faker: Faker): Partial<Product> {
    return {
      type: faker.helpers.enumValue(EProduct), // 👈 'enumValue' does not exist on Helpers
      name: faker.commerce.product(),
      description: faker.commerce.productDescription(),
    };
  }
}

```

### What driver are you using?

@mikro-orm/postgresql

### MikroORM version

5.9.4

### Node.js version

18.17.1

### Operating system

Windows 10

### Validations

- [X] Read the [Contributing Guidelines](https://github.com/mikro-orm/mikro-orm/blob/master/CONTRIBUTING.md).
- [X] Read the [docs](https://mikro-orm.io/docs).
- [X] Check that there isn't [already an issue](https://github.com/mikro-orm/mikro-orm/issues) that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/mikro-orm/mikro-orm/discussions) or join our [Slack](https://join.slack.com/t/mikroorm/shared_invite/enQtNTM1ODYzMzM4MDk3LWM4ZDExMjU5ZDhmNjA2MmM3MWMwZmExNjhhNDdiYTMwNWM0MGY5ZTE3ZjkyZTMzOWExNDgyYmMzNDE1NDI5NjA).
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by B4nan** [comment]
As you already found out, the dependency is removed in v6, if you want to use faker, you need to install it yourself - and you can do the very same thing in v5 too, just ignore the faker instance provided by the ORM. You should be as well able to use [npm overrides](https://docs.npmjs.com/cli/v9/configuring-npm/package-json#overrides) (or its alternatives in other package managers).

I'd rather not update the version in 5.x, as it feels breaking given they changed some APIs.
